src/core/unit.c

```diff
@@ -1699,7 +1699,7 @@ static void unit_status_log_starting_stopping_reloading(Unit *u, JobType t) {
         format = unit_get_status_message_format(u, t);
 
         DISABLE_WARNING_FORMAT_NONLITERAL;
-        xsprintf(buf, format, unit_description(u));
+        snprintf(buf, sizeof buf, format, unit_description(u));
```

Comment:
@keszybz, thank you. I think it would make sense to put `(void)` in front of `sprintf` to prevent `xsprintf.cocci` from patching up everything again. It would also be great not to rely entirely on the thing that is easy to forget about, but @poettering doesn't seem to like https://github.com/systemd/systemd/pull/8210 very much.

Reply:
Yeah, that would be good. I changed most of the other places to `snprintf_ok`, but this one needs a `(void)`.